CHANGELOG.md

```diff
@@ -2,9 +2,13 @@
 
 ## 0.8.2 — [unreleased]
 
+_The following unreleased version are only available in the `main` branch. They
+will become the public version at the next release._
+
 **Features**:
 
-- Add a `~=` regex search operator (@max-sixty, #2458). An example:
+- Add a `~=` regex search operator (@max-sixty,
+  https://github.com/PRQL/prql/pull/2458). An example:
```

Comment:
GitHub will make these automatically. Though they don't show in https://prql-lang.org/book/changelog.html admittedly.

Obv fine to have them here for now, but to keep consistent with the remainder of the doc, I'll probably switch to the existing convention before publishing.

In the short term, we could have a sentence at the top pointing folks towards https://github.com/PRQL/prql/releases, where these are formatted.

In the medium term, we could add this as a preprocessor (or maybe one exists out there?) to make these into links.

Reply:
You're right - I switched it back.